keras_cv/models/object_detection/retinanet/feature_pyramid.py

```diff
@@ -49,10 +49,4 @@ def call(self, inputs, training=False):
         p5_output = self.conv_c5_3x3(p5_output, training=training)
         p6_output = self.conv_c6_3x3(c5_output, training=training)
         p7_output = self.conv_c7_3x3(tf.nn.relu(p6_output), training=training)
-        return {
-            "P3": p3_output,
-            "P4": p4_output,
-            "P5": p5_output,
-            "P6": p6_output,
-            "P7": p7_output,
-        }
+        return p3_output, p4_output, p5_output, p6_output, p7_output
```

Comment:
What is the purpose of switching from dictionary to tuple format here?

Reply:
Because otherwise we're hardcoding the levels output by the FPN for no obvious payoff. There's a lot more flexibility here.

Reply:
In the RetinaNet class' "__init__" we were iterating over the entire dictionary created by the FeaturePyramid class to attach the prediction heads to the FPN, so @jbischof suggested to simplify the contract users will have to adhere to, while also "cleaning" the code. I think it makes sense this way, but the last word is on you guys.